static/scss/dashboard.scss

```diff
@@ -287,3 +287,15 @@ c.validation-wrapper {
 .header-privacy-tab {
   color: #A2AAB3;
 }
+
+button.btn-apply-for-ms {
+  font-family: "Open Sans";
+  background: #B2C9D2;
```

Comment:
This color is only for the disabled state. I know we don't show the button enabled yet, but doesn't it make sense to put this in the disabled style?